### PR TITLE
BD-2691: Clarify how to send only deltas for CDI

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

Closes #**BD-2691**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

### CODE REVIEW

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/overview.md [hunk 1]
```diff
@@ -82,6 +82,162 @@ During the next scheduled sync, all rows with a `UPDATED_AT` timestamp later tha
 }
 ```
 
+### Example: First time sync and subsequent updates
+
+This example shows the general process for syncing data for the first time, then only updating changing data (deltas) in the subsequent updates. Let's say we have a table `EXAMPLE_DATA` with some user data. On day 1, it has the following values:
+
+<style type="text/css">
```
Comment (jayhahn) [inline _docs/_user_guide/data_and_analytics/cloud_ingestion/overview.md:89]: Not too fmiliar with the process of including CSS in the doc files but is this expected? Will it properly be hidden from the view once it's deployed? 

image

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/overview.md [hunk 2]
```diff
@@ -82,6 +82,162 @@ During the next scheduled sync, all rows with a `UPDATED_AT` timestamp later tha
...
+| 2023-03-16 15:00:00 | 12345       | { "ATTRIBUTE_1": "823", "ATTRIBUTE_2":"blue", "ATTRIBUTE_3":"380", "ATTRIBUTE_4":"FALSE"} |
+| 2023-03-16 15:00:00 | 23456       | { "ATTRIBUTE_1": "28", "ATTRIBUTE_2":"blue", "ATTRIBUTE_3":"823", "ATTRIBUTE_4":"TRUE"}   |
+| 2023-03-16 15:00:00 | 34567       | { "ATTRIBUTE_1": "234", "ATTRIBUTE_2":"blue", "ATTRIBUTE_3":"384", "ATTRIBUTE_4":"TRUE"}  |
+| 2023-03-16 15:00:00 | 45678       | { "ATTRIBUTE_1": "245", "ATTRIBUTE_2":"red", "ATTRIBUTE_3":"349", "ATTRIBUTE_4":"TRUE"}   |
+| 2023-03-16 15:00:00 | 56789       | { "ATTRIBUTE_1": "1938", "ATTRIBUTE_2":"red", "ATTRIBUTE_3":"813", "ATTRIBUTE_4":"FALSE"} |
+| 2023-03-16 15:00:00 | 12345       | { "ATTRIBUTE_1": "145", "ATTRIBUTE_2":"red", "ATTRIBUTE_4":"TRUE"} |
+| 2023-03-16 15:00:00 | 23456       | { "ATTRIBUTE_1": "15"} |
+| 2023-03-16 15:00:00 | 34567       | { "ATTRIBUTE_3":"495", "ATTRIBUTE_4":"FALSE"} |
+| 2023-03-16 15:00:00 | 45678       | { "ATTRIBUTE_2":"green"} |
+| 2023-03-16 15:00:00 | 56789       | { "ATTRIBUTE_3":"693"} |
```
Comment (jayhahn) [inline _docs/_user_guide/data_and_analytics/cloud_ingestion/overview.md:236]: Should probably use 2023-03-17 date (and with different time) to note that these the "new rows from morning of day 2"

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/overview.md (hunk 1 above)
Comment (bre-fitzgerald) [reply _docs/_user_guide/data_and_analytics/cloud_ingestion/overview.md:89]: Hey! Are you viewing this via the preview in VSCode? It won't render the CSS in the editor preview. You can go here to see the staged deploy: https://braze-docs-lfaovis59-braze.vercel.app/docs/user_guide/data_and_analytics/cloud_ingestion/overview/#example-first-time-sync-and-subsequent-updates

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/overview.md (hunk 2 above)
Comment (bre-fitzgerald) [reply _docs/_user_guide/data_and_analytics/cloud_ingestion/overview.md:236]: Good catch, thank you! Will update that.

Comment (bre-fitzgerald) [inline _docs/_user_guide/data_and_analytics/cloud_ingestion/overview.md:236]:
```suggestion
| 2023-03-17 17:30:00 | 12345       | { "ATTRIBUTE_1": "145", "ATTRIBUTE_2":"red", "ATTRIBUTE_4":"TRUE"} |
| 2023-03-17 17:30:00 | 23456       | { "ATTRIBUTE_1": "15"} |
| 2023-03-17 17:30:00 | 34567       | { "ATTRIBUTE_3":"495", "ATTRIBUTE_4":"FALSE"} |
| 2023-03-17 17:30:00 | 45678       | { "ATTRIBUTE_2":"green"} |
| 2023-03-17 17:30:00 | 56789       | { "ATTRIBUTE_3":"693"} |
```

**Pull request merged**